code/TestAutomation.Core/Browser/WebDriverExtensions.cs

```diff
@@ -0,0 +1,223 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+using TestAutomation.Core.Elements;
+using TestAutomation.Utilities;
+
+namespace TestAutomation.Core.Browser
```

Comment:
Implement Singleton pattern to guarantee that only one instance of driver will be created.